core/src/main/scala/transport/netty/ClientConnectionPool.scala

```diff
@@ -77,6 +78,7 @@ class NettyConnectionPool(hosts: Process[Task,InetSocketAddress],
         }
       }
       else {
+        pipe.removeLast()
         channel.close()
```

Comment:
@jedesah `channel.close` bothers me a bit as it returns a `ChannelFuture` that is not handled in the same way as:
 ```scala
        Task.async[Channel] { cb =>
          val _ = fut.addListener(new ChannelFutureListener {
                            def operationComplete(cf: ChannelFuture): Unit = {
                              if(cf.isSuccess) { (\/-(cf.channel)) } else { cb(-\/(cf.cause)) }
                            }
                        })
        }
```
at https://github.com/ahjohannessen/remotely/blob/wip-improve-ci-bump-versions/core/src/main/scala/transport/netty/ClientConnectionPool.scala#L258